framework/helpers/BaseHtml.php

```diff
@@ -929,8 +929,8 @@ public static function checkboxList($name, $selection = null, $items = [], $opti
      * @param array $options options (name => config) for the radio button list container tag.
      * The following options are specially handled:
      *
-     * - tag: string|boolean, the tag name of the container element. Disable the container of the
-     *   radio buttons if it is false.
+     * - tag: string|false, string is the tag name of the container element. Disable the container of the radio buttons
```

Comment:
I'm suggesting another description: `False to render radio buttons without container`